source/operations/install-deploy-manage/expand-minio-deployment.rst

```diff
@@ -28,6 +28,24 @@ The procedure on this page expands an existing
 :ref:`distributed <deploy-minio-distributed>` MinIO deployment with an
 additional server pool. 
 
+Rebalancing Pools
+-----------------
+
+MinIO optimizes storage of objects across available pools by writing new objects to the server pool with the most free space compared total amount of free space on all available server pools.
+For the calculations and more details, see :ref:`Writing Files <minio-writing-files>` below.
+As a result, MinIO does not perform the costly action of rebalancing objects from older pools to newer pools.
+Based on write preference calculation, new writes default to the new pool as it has the most free space compared to the other pools until the new pool reaches a similar level of contents compared to the other pools.
+
+Rebalancing data across all pools after an expansion is an expensive operation that requires scanning the entire deployment and moving objects between pools.
+This may take a long time to complete depending on the amount of data to move.
+
+Rebalancing runs in parallel with other processes, though MinIO recommends only running a rebalance operation when the deployment has the available resources.
+Stop and start the rebalance as needed to take advantage of low use periods.
+Write operations during a rebalance go to server pools other than the pool MinIO actively rebalances.
```

Comment:
Write operations on new objects - overwrites on existing objects work as expected.
I wonder if we can simplify this graf to something like:

"Rebalancing does not block ongoing operations and runs in parallel to all other IO. This can result in reduced performance of regular operations. Consider scheduling rebalancing operations during non-peak periods to avoid impacting production workloads. You can start and stop rebalancing at any time"

The specifics of how rebalancing handles IO is probably unnecessary at this time.

Reply:
Took your rewrite as is to replace the graph.